backend/de.metas.handlingunits.base/src/main/sql/postgresql/system/5746380_sys_gh20054_ManufacturingOrderUnprocessedCandidates.sql

```diff
@@ -0,0 +1,19 @@
+-- Run mode: SWING_CLIENT
+
+-- Value: ManufacturingOrderUnprocessedCandidates
+-- 2025-02-12T12:20:45.065Z
+INSERT INTO AD_Message (AD_Client_ID,AD_Message_ID,AD_Org_ID,Created,CreatedBy,EntityType,IsActive,MsgText,MsgType,Updated,UpdatedBy,Value) VALUES (0,545497,0,TO_TIMESTAMP('2025-02-12 12:20:44.803000','YYYY-MM-DD HH24:MI:SS.US')::timestamp without time zone AT TIME ZONE 'UTC',100,'D','Y','Der Produktionsauftrag wurde noch nicht endgültig bearbeitet. Bitte bearbeiten Sie die offenen Posten, bevor Sie den Auftrag abschließen.','E',TO_TIMESTAMP('2025-02-12 12:20:44.803000','YYYY-MM-DD HH24:MI:SS.US')::timestamp without time zone AT TIME ZONE 'UTC',100,'ManufacturingOrderUnprocessedCandidates')
```

Comment:
de_DE: `Bitte führen Sie die Aktion "Verarbeiten" im "Produzieren"-Dialog aus, bevor Sie den Produktionsauftrag abschließen.`
de_CH: `Bitte führen Sie die Aktion "Verarbeiten" im "Produzieren"-Dialog aus, bevor Sie den Produktionsauftrag abschliessen.`
de_EN: `Please execute the "Process" action in the "Issue/Receipt"-Dialog before closing this manufacturing order.`